androidapp/app/build.gradle

```diff
@@ -45,8 +45,13 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
     implementation 'androidx.work:work-runtime-ktx:2.3.4'
     implementation 'com.google.code.gson:gson:2.8.5'
```

Comment:
Can we remove Gson now?

- [ ] Remove Gson if necessary

Reply:
Probably